CONTRIBUTING.md

```diff
@@ -61,6 +61,21 @@ react-native init --version ${RN_VERSION}
 npm config set registry https://registry.npmjs.org/
 ```
 
+## Running RN using local version of CLI
+
+To start a new project with local version of CLI and RN without publishing or proxy, you can follow these steps:
+
+1. Check out `react-native` repo. Then update template in local `react-native/template/package.json`, replacing dependency version of `react-native` with the absolute path of the react native repo, for example: "react-native": "file:///Users/username/react-native"
+
+1. Check out `react-native-cli` repo, cd into it and run `yarn` and `yarn build`.
```

Comment:
This step is redundant, since we advise to o it at the very beginning of this guide. Also `yarn build` runs on postinstall hook, so it's not necessary as well